tests/_data/index.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+use Psr\Container\ContainerInterface;
+use Yiisoft\Composer\Config\Builder;
+use Yiisoft\Di\Container;
+use Yiisoft\Http\Method;
+use Yiisoft\Yii\Web\Application;
+use Yiisoft\Yii\Web\Config\EventConfigurator;
+use Yiisoft\Yii\Web\SapiEmitter;
+use Yiisoft\Yii\Web\ServerRequestFactory;
+
+$c3 = dirname(__dir__, 2) . '/c3.php';
+$autoload = dirname(__dir__, 2) . '/vendor/autoload.php';
```

Comment:
Is it the same as regular index except c3? If so, maybe we can do `require` instead of copying it?

Reply:
done.